skaffold-plugins-core/src/test/java/com/google/cloud/tools/skaffold/command/SkaffoldTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2018 Google LLC. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.skaffold.command;
+
+import com.google.cloud.tools.skaffold.filesystem.FilePermissions;
+import com.google.cloud.tools.skaffold.filesystem.OperatingSystem;
+import com.google.common.io.Resources;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/** Tests for {@link Skaffold}. */
+@RunWith(MockitoJUnitRunner.class)
+public class SkaffoldTest {
+
+  private static void verifyDeploy(String expectedStdout, String expectedStderr, String... command)
+      throws InterruptedException, ExecutionException, IOException {
+    InputStream stdinInputStream =
+        new ByteArrayInputStream("input".getBytes(StandardCharsets.UTF_8));
```

Comment:
"input" seems odd?  Ahh! Would be helpful to say that that the test executes an script instead of the Skaffold binary which provides known outputs.